### PR TITLE
TET-74 basic page

### PR DESCRIPTION
**Pull request by VadimST04**
- create class TetrahedronApp that consists all components of basic page
- create class Navbar hat consists of logo (image via ImageView and text) and language ComboBox (need to be developed in the future)
- create InstrumentsSidebar that is located on the left side of the page and consists of tools icons

https://github.com/user-attachments/assets/97921da9-620f-404f-99dd-cdc3e501a999

### CODE REVIEW

File: src/main/java/com/tetrahedron/app/htmlApp/ViewScene.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        main.getChildren().addAll(instrumentSidebar, experience);
+
+        root.getChildren().addAll(navbar, main);
+
+        stage.setScene(scene);
+        stage.setTitle("Tetrahedron Model View");
+        stage.show();
+    }
+
+    public static void main(String[] args) {
```
Comment (HappyMary16) [inline src/main/java/com/tetrahedron/app/htmlApp/ViewScene.java:40]: Usually, a file with the main method is located in main package, in our case it is com.tetrahedron.app

File: src/main/java/com/tetrahedron/app/htmlApp/ViewScene.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.Objects;
+
+public class ViewScene extends Application {
```
Comment (HappyMary16) [inline src/main/java/com/tetrahedron/app/htmlApp/ViewScene.java:11]: Maybe lets rename it to TetrahedronApp
TO make it more class that it's a main class of our program

**Review by HappyMary16 — CHANGES_REQUESTED** [review]

**Comment by HappyMary16** [comment]
Could you please add a description to the pill request? And it would be great if you can attach a video with a result :)

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <junit.version>5.9.2</junit.version>
+        <javafx.maven.plugin.version>0.0.8</javafx.maven.plugin.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-controls</artifactId>
+            <version>21-ea+24</version>
```
[the remark this answers is not in the document]
Comment (VadimST04) [reply pom.xml:22]: Yes, I will try to implement it

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-fxml</artifactId>
+            <version>21-ea+24</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
```
[the remark this answers is not in the document]
Comment (VadimST04) [reply pom.xml:32]: Theses were some default dependencies. I can delete them. Maybe we will write unit tests, but not now :)

[the remark this answers is not in the document]
Comment (HappyMary16) [reply pom.xml:33]: If they are not used now, let's delete them.
If we will need them later, we will add them later

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (HappyMary16) [reply pom.xml]: Looks like java fx doesn't have anything like this :( So versions specified in properties section is ok

**Review by GurgenAvagyan — APPROVED** [review]

**Comment by HappyMary16** [comment]
Looks like htmlApp folder needs to be deleted. All classes inside are duplicated.

**Review by HappyMary16 — CHANGES_REQUESTED** [review]
Looks like htmlApp folder needs to be deleted. All classes inside are duplicated.

File: src/main/java/com/tetrahedron/app/htmlApp/InstrumentsSidebar.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+package com.tetrahedron.app.htmlApp;
```
Comment (HappyMary16) [inline src/main/java/com/tetrahedron/app/htmlApp/InstrumentsSidebar.java:1]: Why do files in this folder still exist? Do we need them?

**Review by HappyMary16 — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 6]
```diff
@@ -1,98 +1,85 @@
...
+            <version>${junit.version}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.lwjgl</groupId>
-            <artifactId>lwjgl-glfw</artifactId>
-            <version>${lwjgl.version}</version>
-            <classifier>natives-${platform}</classifier>
+            <groupId>org.junit.jupiter</groupId>
```
Comment (HappyMary16) [inline pom.xml:38]: why do we still have this dependency for unit tests? you need to remove them or add unit tests

File: pom.xml [hunk 7]
```diff
@@ -1,98 +1,85 @@
...
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
-        <joml.version>1.10.8</joml.version>
-        <lwjgl.version>3.3.4</lwjgl.version>
+        <junit.version>5.9.2</junit.version>
+        <javafx.version>24-ea+5</javafx.version>
+        <javafx.maven.plugin.version>0.0.8</javafx.maven.plugin.version>
     </properties>
 
-    <profiles>
```
Comment (HappyMary16) [inline pom.xml:20]: Is lwjgl version working with all these libraries removed?
both versions of the program must work 
I'm 100% percent sure that they may coexist

File: .gitignore [hunk 8]
```diff
@@ -9,7 +9,6 @@ build/
 .idea/jarRepositories.xml
 .idea/compiler.xml
 .idea/libraries/
-!**target/**
```
[the remark this answers is not in the document]
Comment (VadimST04) [reply .gitignore:12]: No, we don't need it in git repo, so I added it again

File: .gitignore [hunk 9]
```diff
@@ -40,6 +39,4 @@ bin/
 .vscode/
 
 ### Mac OS ###
-.DS_Store
-/.idea/
```
[the remark this answers is not in the document]
Comment (VadimST04) [reply .gitignore:44]: added them too

File: pom.xml (hunk 7 above)
Comment (VadimST04) [reply pom.xml:20]: Yes, you are right, I combined two pom files

**Review by HappyMary16 — APPROVED** [review]

**Pull request merged**